packages/utils/src/TestUtils.ts

```diff
@@ -52,6 +52,24 @@ class TestUtils {
     fn: (...args: TArgs) => TResult
   ): jest.Mock<TResult, TArgs> => (fn as unknown) as jest.Mock<TResult, TArgs>;
 
+  /**
+   * Selectively disable logging methods on `console` object. Uses spyOn so that
+   * changes will be reverted after leaving the test scope that it is set in. If
+   * no method names are given, all will be disabled.
+   */
+  static disableConsoleOutput = (
+    ...methodNames: ('log' | 'warn' | 'error' | 'info' | 'debug')[]
+  ): void => {
+    if (methodNames.length === 0) {
+      // eslint-disable-next-line no-param-reassign
+      methodNames = ['log', 'warn', 'error', 'info', 'debug'];
```

Comment:
Might as well mock everything?

```suggestion
      methodNames = Object.getOwnPropertyNames(console).filter(name => typeof console[name] === 'function');
```

Reply:
agreed